src/Adapter/Product/Repository/ProductRepository.php

```diff
@@ -168,6 +171,7 @@ public function get(ProductId $productId): Product
             Product::class,
             ProductNotFoundException::class
         );
+        $product->loadStockData();
```

Comment:
should we wrap this in try-catch? (replacing the PrestaShopException to CoreException)?
I know we have not covered this 100%, but Its either we do it everywhere or don't bother at all :smile: (I would go for not bothering, but I've been told to do it since first days of my work in migration :sweat:)

Reply:
Yeah you're right, nice catch!